backend/services/implementations/__tests__/testService.test.ts

```diff
@@ -43,4 +44,20 @@ describe("mongo testService", (): void => {
       await testService.createTest(mockTest);
     }).rejects.toThrowError(`userId ${mockTest.admin} not found`);
   });
-});
+
+
+  it("getTestById", async () => {
+    userService.getUserById = jest.fn().mockReturnValue(mockAdmin);
+    const test = await testService.createTest(mockTest);
+    const res = await testService.getTestById(test.id);
+
+    expect(res).toMatchObject(test);
```

Comment:
this was erroring out because `.toMatchObject()` doesn't work for the nested values

Reply:
ohhh i see, tysm!!